msteams-platform/concepts/device-capabilities/people-picker-capability.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Integrate People Picker capability
+author: Rajeshwari-v
+description: How to use Teams JavaScript client SDK to integrate People Picker capability
+keywords:  people picker control
+ms.topic: conceptual
+localization_priority: Normal
+ms.author: surbhigupta
+---
+
+# Integrate People Picker capability 
+
+People Picker is a control to search and select people. This is a native capability available in Teams platform. You can integrate Teams’ native People Picker input control with your web apps. You can select between single or multi-selection, and configurations such as limiting search within a chat/channels, or across the entire organization.
+
+You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides `People Picker` API to integrate the People Picker capability within your web app. 
+
+## Advantages of integrating People Picker capability
+
+* The People Picker control works in all of Teams surfaces, such as task module, a chat, channel, meeting tab, and personal app.
+* This cross platform control allows users to select entities, such as users in a group, chat, channel.
+* The People Picker capability helps with task assignment, tagging, notifying a user, and so on. 
+
+You must call the `People Picker` API to integrate People Picker capability. For effective integration, you must have an understanding of [code snippet](#code-snippet) for calling the `People Picker` API. 
+It is important to familiarize yourself with the [API response errors](#error-handling) to handle the errors in your web app.
+
+> [!NOTE] 
+> Currently, Microsoft Teams support for People Picker capability is available for mobile clients only.
+
+## People Picker API 
+
+`People Picker` API enables you to add Teams’ native `People Picker input control` to your web apps. This API returns the information of picked users to the calling web app. 
+You must use the following API to enable People Picker capability:
+
+| API      | Description   |
+| --- | --- |
+|`selectPeople`|Launches a People Picker and allows the user to select one or more people from the list.<br/><br/>If the app is added to personal app scope, the People Picker is launched across the organization. If the app is added to a chat or channel, People Picker launch is limited to the members of that chat or channel.|
```

Comment:
```suggestion
|`selectPeople`|Launches a People Picker and allows the user to search and select one or more people from the list.<br/><br/>In case of a personal app the control searches across the organization. If the app is added to a chat or channel, then the search context can be configured depending on the scenario. The search can be restricted within the members of that chat or channel, or can be made organization-wide as well.|
```